homeassistant/components/switcher_kis/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration uses entity services.
+  appropriate-polling:
+    status: exempt
+    comment: The integration does not poll.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+    status: todo
+    comment: make sure flows end with created entry or abort
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup:
+    status: exempt
+    comment: devices are setup asynchronously and marked as unavailable until they are ready.
+  unique-config-entry:
+    status: exempt
+    comment: The integration only supports a single config entry.
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: todo
+  reauthentication-flow: done
+  test-coverage: done
```

Comment:
It would be nice if we can phase out monkeypatch

Reply:
Where is it documented that this is a requirement? I agree about phasing this out, but I don't see this as a mandatory for tests coverage.